library/core/src/mem/manually_drop.rs

```diff
@@ -116,8 +116,8 @@ impl<T> ManuallyDrop<T> {
     /// leaving the state of this container unchanged.
     /// It is your responsibility to ensure that this `ManuallyDrop` is not used again.
     ///
-    /// [`ManuallyDrop::drop`]: #method.drop
-    /// [`ManuallyDrop::into_inner`]: #method.into_inner
+    /// [`drop`]: ManuallyDrop::drop
+    /// [`into_inner`]: ManuallyDrop::into_inner
```

Comment:
I don't see anything using `drop` or `into_inner` so you can remove these reference links :)
```suggestion
```